### ISSUE TITLE
What is ... (more hidden) ...?

### ISSUE THREAD

**Issue by soraxas** [tqdm/tqdm#963]
Hi! 

I want to ask what is this for?
https://github.com/tqdm/tqdm/blob/89ee14464b963ea9ed8dafe64a2dad8586cf9a29/tqdm/std.py#L1467
I've searched on issues/online and haven't found anything related to this.

I first encountered this when I was running experiment in tmux, and I thought it was tmux that was suppressing output. I tried to search online but couldn't found anything related to it, and turns out the phrase `... (more hidden) ...` comes directly from `tqdm`'s source code.

It became problematic to me because after running my python script for a while (with multiple panels and each script repeating multiple times), most panels starts to show this message, and stops updating (i.e. tmux's panel became blank, no progress bar) even though the script may (or may not) be still running. Once I terminates it (with Ctrl-C) the tmux panel may flush out multiple progress bar that were presents (but never shown in terminal stdout), along with the trackback caused by Ctrl-C.

I am not sure what this line is trying to do, and my problem might as well be tmux's panel being filled by buffer? I just want to understand what this message `... (more hidden) ...` is for.

Thanks! :)

**Comment by casperdcl** [comment]
If there are more nested bars than screen height, this message should display at the bottom of the screen. It could be that you're manually positioning bars off-screen so there's a long period where everything looks blank.

**Comment by soraxas** [comment]
Hi @casperdcl thank you very much for your swift reply!

I understand that and it is very reasonable. The first thing that sparks me is that I wasn't using nested bar nor am I manually positioning the bar. After digging around in the source code and testing different scenario, I think I (probably?) may had found the reason of my misuse.

I was using `pbar = tqdm(...)` and never closed the pbar with `pbar.close()` after the loop. When I was running my experiment I was repeating the entire experiment multiple times, and I think previously I was relying on python scope to clean things up. However, with `tqdm`'s auto nesting bar features, (where it uses `cls._instances = WeakSet()` to keep track of initialised pbar), it tries to display the progress bar as a nested bar if the **weak ref** hasn't cleaned up the old progress bar yet. Therefore, it tries to display progress bar at `nrow` that is much larger than screen size. And that is also why the problem that I described sometimes happens but sometimes not (because it depends on garbage collector).

#### tl;dr: 
1. I never called `pbar.close()`, hence even if `pbar` went out of scope, if garbage collect hasn't decided to clean it up, `cls._instances` will holds references to **out of scoped pbar**
2. Let's say if `len(cls._instances) = 10` where items within are objects without strong references, the actual **active** pbar will tries to print at `pos=10`, and in my case tmux's panel are quite tiny so the entire screen went blank (blank because the other `pbar` aren't actually active and not printing anything) with ends up with only `... (more hidden) ...` at the bottom.

All in all, cleans up after your usage of `pbar` :) Thanks for taking your time to explain the code @casperdcl

**Comment by casperdcl** [comment]
> using `pbar = tqdm(...)` and never closed the pbar with `pbar.close()` after the loop

you could also do

```python
with tqdm(...) as pbar:
   ...
```

to avoid having to remember to `close()`
